src/amo/components/HomeHeroGuides/styles.scss

```diff
@@ -58,12 +58,16 @@ $icon-size: 40px;
       padding: 0;
       text-decoration: none;
 
-      &:focus .HeroSection-content {
+      &.focus-visible .HeroSection-content {
         // Fix outline for Firefox
         outline: 1px dotted $link-color;
         // Restore default outline for webkit.
         outline: auto 5px -webkit-focus-ring-color;
       }
+
+      &:not(.focus-visible) .HeroSection-content {
```

Comment:
Would it work if this was the default and only `&.focus-visible` is added to override this default state?

Reply:
do you mean by "default" put this before the `&.focus-visible .HeroSection-content` styles? or put this `&:focus .HeroSection-content`?